commit 1fc2a47f0b6
Author: Sam Elliott <selliott@lowrisc.org>
Date:   Thu Aug 1 14:23:56 2019 +0000

    Add support for openSUSE RISC-V triple
    
    Reviewers: asb
    
    Reviewed By: asb
    
    Subscribers: lenary, rbar, johnrusso, simoncook, apazos, sabuasal, niosHD, jrtc27, zzheng, edward-jones, MartinMosbeck, brucehoult, the_o, PkmX, jocewei, lebedev.ri, kito-cheng, shiva0217, rogfer01, dexonsmith, rkruppe, cfe-commits, llvm-commits
    
    Tags: #clang, #llvm
    
    Differential Revision: https://reviews.llvm.org/D63497
    
    Patch by Andreas Schwab (schwab)
    
    llvm-svn: 367565

Index: clang-9.0.1.src/lib/Driver/ToolChains/Gnu.cpp
===================================================================
--- clang-9.0.1.src.orig/lib/Driver/ToolChains/Gnu.cpp
+++ clang-9.0.1.src/lib/Driver/ToolChains/Gnu.cpp
@@ -2018,7 +2018,8 @@ void Generic_GCC::GCCInstallationDetecto
   static const char *const RISCV64LibDirs[] = {"/lib64", "/lib"};
   static const char *const RISCV64Triples[] = {"riscv64-unknown-linux-gnu",
                                                "riscv64-linux-gnu",
-                                               "riscv64-unknown-elf"};
+                                               "riscv64-unknown-elf",
+                                               "riscv64-suse-linux"};
 
   static const char *const SPARCv8LibDirs[] = {"/lib32", "/lib"};
   static const char *const SPARCv8Triples[] = {"sparc-linux-gnu",
